Can't run latest master

I wanted to switch to master from last stable release but got this error:
![2017-08-08-142537_1320x1409_scrot](https://user-images.githubusercontent.com/163128/29071818-74401568-7c45-11e7-97ba-b5e2213cb7be.png)

terminal output:
```sh
oni-master > npm run start

> oni@0.2.8 start /home/hoschi/repos/oni-master
> concurrently --kill-others "npm run start-hot" "npm run watch:browser" "npm run watch:plugins"

[0] 
[0] > oni@0.2.8 start-hot /home/hoschi/repos/oni-master
[0] > cross-env NODE_ENV=development electron main.js
[0] 
[1] 
[1] > oni@0.2.8 watch:browser /home/hoschi/repos/oni-master
[1] > webpack-dev-server --config browser/webpack.hotreload.config.js --inline --hot --host localhost --port 8191
[1] 
[2] 
[2] > oni@0.2.8 watch:plugins /home/hoschi/repos/oni-master
[2] > cd vim/core/oni-plugin-typescript && tsc --watch
[2] 
[0] Added extension: Redux DevTools
[1]  http://localhost:8191/
[1] webpack result is served from http://localhost:8191/
[1] content is served from ./dist
[1] webpack: wait until bundle finished: /bundle.js
[1] ts-loader: Using typescript@2.2.1 and /home/hoschi/repos/oni-master/browser/tsconfig.src.json
[2] src/LiveEvaluation.ts(34,16): error TS2693: 'Promise' only refers to a type, but is being used as a value here.
[2] src/LiveEvaluation.ts(94,16): error TS2693: 'Promise' only refers to a type, but is being used as a value here.
[2] src/TypeScriptServerHost.ts(243,29): error TS2693: 'Promise' only refers to a type, but is being used as a value here.
[2] src/index.ts(165,20): error TS2693: 'Promise' only refers to a type, but is being used as a value here.
[2] src/index.ts(187,20): error TS2693: 'Promise' only refers to a type, but is being used as a value here.
[2] src/index.ts(210,20): error TS2693: 'Promise' only refers to a type, but is being used as a value here.
[2] 14:23:53 - Compilation complete. Watching for file changes.
[2] 
[2] 
[1] Hash: 771fdc0426febf3fb0a8
[1] Version: webpack 1.13.2
[1] Time: 6810ms
[1]         Asset     Size  Chunks             Chunk Names
[1]     bundle.js  2.59 MB       0  [emitted]  main
[1] bundle.js.map   3.2 MB       0  [emitted]  main
[1] chunk    {0} bundle.js, bundle.js.map (main) 2.48 MB [rendered]
[1]     [0] multi main 52 bytes {0} [built]
[1]     [2] (webpack)-dev-server/client?http://localhost:8191 3.97 kB {0} [built]
[1]     [4] ./~/strip-ansi/index.js 161 bytes {0} [built]
[1]     [5] ./~/ansi-regex/index.js 135 bytes {0} [built]
[1]     [6] (webpack)-dev-server/client/socket.js 856 bytes {0} [built]
[1]     [7] ./~/sockjs-client/lib/entry.js 244 bytes {0} [built]
[1]     [8] ./~/sockjs-client/lib/transport-list.js 613 bytes {0} [built]
[1]     [9] ./~/sockjs-client/lib/transport/websocket.js 2.72 kB {0} [built]
[1]    [10] ./~/sockjs-client/lib/utils/event.js 2 kB {0} [built]
[1]    [11] ./~/sockjs-client/lib/utils/random.js 746 bytes {0} [built]
[1]    [13] ./~/sockjs-client/lib/utils/url.js 975 bytes {0} [built]
[1]    [14] ./~/url-parse/index.js 11.4 kB {0} [built]
[1]    [15] ./~/requires-port/index.js 753 bytes {0} [built]
[1]    [16] ./~/url-parse/~/querystringify/index.js 1.51 kB {0} [built]
[1]    [17] ./~/debug/src/index.js 263 bytes {0} [built]
[1]    [18] ./~/debug/src/browser.js 4.73 kB {0} [built]
[1]    [19] ./~/debug/src/debug.js 4.39 kB {0} [built]
[1]    [20] ./~/ms/index.js 2.76 kB {0} [built]
[1]    [21] ./~/debug/src/node.js 5.97 kB {0} [built]
[1]    [26] ./~/inherits/inherits.js 192 bytes {0} [built]
[1]    [27] ./~/inherits/inherits_browser.js 672 bytes {0} [built]
[1]    [29] ./~/sockjs-client/lib/transport/driver/websocket.js 51 bytes {0} [built]
[1]    [30] ./~/sockjs-client/~/faye-websocket/lib/faye/websocket.js 1.3 kB {0} [built]
[1]    [31] ./~/websocket-driver/lib/websocket/driver.js 1.39 kB {0} [built]
[1]    [32] ./~/websocket-driver/lib/websocket/driver/base.js 3.47 kB {0} [built]
[1]    [33] ./~/websocket-driver/lib/websocket/streams.js 4.91 kB {0} [built]
[1]    [35] ./~/websocket-driver/lib/websocket/driver/headers.js 815 bytes {0} [built]
[1]    [36] ./~/websocket-driver/lib/websocket/driver/stream_reader.js 1.87 kB {0} [built]
[1]    [37] ./~/websocket-driver/lib/websocket/driver/client.js 4.16 kB {0} [built]
[1]    [38] ./~/websocket-driver/lib/websocket/http_parser.js 2.5 kB {0} [built]
[1]    [39] ./~/websocket-driver/lib/websocket/driver/hybi.js 14.1 kB {0} [built]
[1]    [40] ./~/websocket-extensions/lib/websocket_extensions.js 4.86 kB {0} [built]
[1]    [41] ./~/websocket-extensions/lib/parser.js 2.84 kB {0} [built]
[1]    [42] ./~/websocket-extensions/lib/pipeline/index.js 1.48 kB {0} [built]
[1]    [43] ./~/websocket-extensions/lib/pipeline/cell.js 1.42 kB {0} [built]
[1]    [44] ./~/websocket-extensions/lib/pipeline/functor.js 1.35 kB {0} [built]
[1]    [45] ./~/websocket-extensions/lib/pipeline/ring_buffer.js 1.6 kB {0} [built]
[1]    [46] ./~/websocket-extensions/lib/pipeline/pledge.js 779 bytes {0} [built]
[1]    [47] ./~/websocket-driver/lib/websocket/driver/hybi/frame.js 373 bytes {0} [built]
[1]    [48] ./~/websocket-driver/lib/websocket/driver/hybi/message.js 888 bytes {0} [built]
[1]    [49] ./~/websocket-driver/lib/websocket/driver/proxy.js 2.62 kB {0} [built]
[1]    [50] ./~/websocket-driver/lib/websocket/driver/server.js 3.09 kB {0} [built]
[1]    [51] ./~/websocket-driver/lib/websocket/driver/draft75.js 3.06 kB {0} [built]
[1]    [52] ./~/websocket-driver/lib/websocket/driver/draft76.js 2.99 kB {0} [built]
[1]    [53] ./~/sockjs-client/~/faye-websocket/lib/faye/websocket/api.js 5.34 kB {0} [built]
[1]    [54] ./~/sockjs-client/~/faye-websocket/lib/faye/websocket/api/event_target.js 624 bytes {0} [built]
[1]    [55] ./~/sockjs-client/~/faye-websocket/lib/faye/websocket/api/event.js 505 bytes {0} [built]
[1]    [56] ./~/sockjs-client/~/faye-websocket/lib/faye/websocket/client.js 2.72 kB {0} [built]
[1]    [58] ./~/sockjs-client/~/faye-websocket/lib/faye/eventsource.js 3.76 kB {0} [built]
[1]    [59] ./~/sockjs-client/lib/transport/xhr-streaming.js 1.25 kB {0} [built]
[1]    [60] ./~/sockjs-client/lib/transport/lib/ajax-based.js 1.31 kB {0} [built]
[1]    [61] ./~/sockjs-client/lib/transport/lib/sender-receiver.js 1.18 kB {0} [built]
[1]    [62] ./~/sockjs-client/lib/transport/lib/buffered-sender.js 2.3 kB {0} [built]
[1]    [63] ./~/sockjs-client/lib/transport/lib/polling.js 1.32 kB {0} [built]
[1]    [64] ./~/sockjs-client/lib/transport/receiver/xhr.js 1.58 kB {0} [built]
[1]    [65] ./~/sockjs-client/lib/transport/sender/xhr-cors.js 343 bytes {0} [built]
[1]    [66] ./~/sockjs-client/lib/transport/driver/xhr.js 1.64 kB {0} [built]
[1]    [69] ./~/sockjs-client/lib/transport/sender/xhr-local.js 352 bytes {0} [built]
[1]    [70] ./~/sockjs-client/lib/utils/browser.js 560 bytes {0} [built]
[1]    [71] ./~/sockjs-client/lib/transport/xdr-streaming.js 984 bytes {0} [built]
[1]    [72] ./~/sockjs-client/lib/transport/sender/xdr.js 2.46 kB {0} [built]
[1]    [73] ./~/sockjs-client/lib/transport/eventsource.js 766 bytes {0} [built]
[1]    [74] ./~/sockjs-client/lib/transport/receiver/eventsource.js 1.58 kB {0} [built]
[1]    [75] ./~/eventsource/lib/eventsource.js 8.9 kB {0} [built]
[1]    [76] ./~/original/index.js 1.04 kB {0} [built]
[1]    [77] ./~/original/~/url-parse/index.js 6.55 kB {0} [built]
[1]    [78] ./~/original/~/url-parse/lolcation.js 1.4 kB {0} [built]
[1]    [79] ./~/querystringify/index.js 1.3 kB {0} [built]
[1]    [80] ./~/sockjs-client/lib/transport/lib/iframe-wrap.js 981 bytes {0} [built]
[1]    [81] ./~/sockjs-client/lib/transport/iframe.js 3.83 kB {0} [built]
[1]    [82] ./~/json3/lib/json3.js 43.3 kB {0} [built]
[1]    [83] (webpack)/buildin/module.js 251 bytes {0} [built]
[1]    [84] (webpack)/buildin/amd-options.js 43 bytes {0} [built]
[1]    [85] ./~/sockjs-client/lib/version.js 26 bytes {0} [built]
[1]    [86] ./~/sockjs-client/lib/utils/iframe.js 5.03 kB {0} [built]
[1]    [87] ./~/sockjs-client/lib/utils/object.js 532 bytes {0} [built]
[1]    [88] ./~/sockjs-client/lib/transport/htmlfile.js 710 bytes {0} [built]
[1]    [89] ./~/sockjs-client/lib/transport/receiver/htmlfile.js 2.2 kB {0} [built]
[1]    [90] ./~/sockjs-client/lib/transport/xhr-polling.js 894 bytes {0} [built]
[1]    [91] ./~/sockjs-client/lib/transport/xdr-polling.js 712 bytes {0} [built]
[1]    [92] ./~/sockjs-client/lib/transport/jsonp-polling.js 1.02 kB {0} [built]
[1]    [93] ./~/sockjs-client/lib/transport/receiver/jsonp.js 5.57 kB {0} [built]
[1]    [94] ./~/sockjs-client/lib/transport/sender/jsonp.js 2.46 kB {0} [built]
[1]    [95] ./~/sockjs-client/lib/main.js 11.9 kB {0} [built]
[1]    [96] ./~/sockjs-client/lib/shims.js 17.2 kB {0} [built]
[1]    [97] ./~/sockjs-client/lib/utils/escape.js 2.36 kB {0} [built]
[1]    [98] ./~/sockjs-client/lib/utils/transport.js 1.35 kB {0} [built]
[1]    [99] ./~/sockjs-client/lib/utils/log.js 450 bytes {0} [built]
[1]   [100] ./~/sockjs-client/lib/event/event.js 477 bytes {0} [built]
[1]   [101] ./~/sockjs-client/lib/event/eventtarget.js 1.85 kB {0} [built]
[1]   [102] ./~/sockjs-client/lib/location.js 177 bytes {0} [built]
[1]   [103] ./~/sockjs-client/lib/event/close.js 295 bytes {0} [built]
[1]   [104] ./~/sockjs-client/lib/event/trans-message.js 292 bytes {0} [built]
[1]   [105] ./~/sockjs-client/lib/info-receiver.js 2.22 kB {0} [built]
[1]   [106] ./~/sockjs-client/lib/transport/sender/xhr-fake.js 456 bytes {0} [built]
[1]   [107] ./~/sockjs-client/lib/info-iframe.js 1.52 kB {0} [built]
[1]   [108] ./~/sockjs-client/lib/info-iframe-receiver.js 791 bytes {0} [built]
[1]   [109] ./~/sockjs-client/lib/info-ajax.js 1.03 kB {0} [built]
[1]   [110] ./~/sockjs-client/lib/iframe-bootstrap.js 2.9 kB {0} [built]
[1]   [111] ./~/sockjs-client/lib/facade.js 723 bytes {0} [built]
[1]   [112] (webpack)/hot/dev-server.js 1.85 kB {0} [built]
[1]   [113] (webpack)/hot/log-apply-result.js 813 bytes {0} [built]
[1]   [114] ./browser/src/index.tsx 4.18 kB {0} [built]
[1]   [115] ./~/react-hot-api/modules/index.js 57 bytes {0} [built]
[1]   [116] ./~/react-hot-api/modules/makeMakeHot.js 1.07 kB {0} [built]
[1]   [117] ./~/react-hot-api/modules/makePatchReactClass.js 1.3 kB {0} [built]
[1]   [118] ./~/react-hot-api/modules/makeAssimilatePrototype.js 1.85 kB {0} [built]
[1]   [119] ./~/react-hot-api/modules/requestForceUpdateAll.js 888 bytes {0} [built]
[1]   [120] ./~/react-hot-api/modules/deepForceUpdate.js 1.23 kB {0} [built]
[1]   [121] ./~/react-hot-api/modules/bindAutoBindMethods.js 2.21 kB {0} [built]
[1]   [122] ./~/react-hot-api/modules/traverseRenderedChildren.js 536 bytes {0} [built]
[1]   [123] ./~/react-hot-loader/RootInstanceProvider.js 1.18 kB {0} [built]
[1]   [124] ./~/react-hot-loader/getRootInstancesFromReactMount.js 209 bytes {0} [built]
[1]   [125] ./~/react-dom/lib/ReactMount.js 25.5 kB {0} [built]
[1]   [126] ./~/react-dom/lib/reactProdInvariant.js 1.24 kB {0} [built]
[1]   [127] ./~/react-dom/lib/DOMLazyTree.js 3.71 kB {0} [built]
[1]   [128] ./~/react-dom/lib/DOMNamespaces.js 505 bytes {0} [built]
[1]   [129] ./~/react-dom/lib/setInnerHTML.js 3.86 kB {0} [built]
[1]   [130] ./~/fbjs/lib/ExecutionEnvironment.js 1.06 kB {0} [built]
[1]   [131] ./~/react-dom/lib/createMicrosoftUnsafeLocalFunction.js 810 bytes {0} [built]
[1]   [132] ./~/react-dom/lib/setTextContent.js 1.45 kB {0} [built]
[1]   [133] ./~/react-dom/lib/escapeTextContentForBrowser.js 3.43 kB {0} [built]
[1]   [134] ./~/react-dom/lib/DOMProperty.js 8.24 kB {0} [built]
[1]   [135] ./~/fbjs/lib/invariant.js 1.63 kB {0} [built]
[1]   [136] ./~/react/lib/React.js 5.08 kB {0} [built]
[1]   [137] ./~/object-assign/index.js 2.11 kB {0} [built]
[1]   [138] ./~/react/lib/ReactBaseClasses.js 5.44 kB {0} [built]
[1]   [139] ./~/react/lib/reactProdInvariant.js 1.24 kB {0} [built]
[1]   [140] ./~/react/lib/ReactNoopUpdateQueue.js 3.36 kB {0} [built]
[1]   [141] ./~/fbjs/lib/warning.js 2.02 kB {0} [built]
[1]   [142] ./~/fbjs/lib/emptyFunction.js 1.08 kB {0} [built]
[1]   [143] ./~/react/lib/canDefineProperty.js 661 bytes {0} [built]
[1]   [144] ./~/fbjs/lib/emptyObject.js 458 bytes {0} [built]
[1]   [145] ./~/react/lib/lowPriorityWarning.js 2.16 kB {0} [built]
[1]   [146] ./~/react/lib/ReactChildren.js 6.19 kB {0} [built]
[1]   [147] ./~/react/lib/PooledClass.js 3.36 kB {0} [built]
[1]   [148] ./~/react/lib/ReactElement.js 11.2 kB {0} [built]
[1]   [149] ./~/react/lib/ReactCurrentOwner.js 621 bytes {0} [built]
[1]   [150] ./~/react/lib/ReactElementSymbol.js 622 bytes {0} [built]
[1]   [151] ./~/react/lib/traverseAllChildren.js 7.03 kB {0} [built]
[1]   [152] ./~/react/lib/getIteratorFn.js 1.12 kB {0} [built]
[1]   [153] ./~/react/lib/KeyEscapeUtils.js 1.29 kB {0} [built]
[1]   [154] ./~/react/lib/ReactDOMFactories.js 5.48 kB {0} [built]
[1]   [155] ./~/react/lib/ReactElementValidator.js 9.2 kB {0} [built]
[1]   [156] ./~/react/lib/ReactComponentTreeHook.js 11.8 kB {0} [built]
[1]   [157] ./~/react/lib/checkReactTypeSpec.js 4.19 kB {0} [built]
[1]   [158] ./~/react/lib/ReactPropTypeLocationNames.js 572 bytes {0} [built]
[1]   [159] ./~/react/lib/ReactPropTypesSecret.js 442 bytes {0} [built]
[1]   [160] ./~/react/lib/ReactPropTypes.js 500 bytes {0} [built]
[1]   [161] ./~/prop-types/factory.js 890 bytes {0} [built]
[1]   [162] ./~/prop-types/factoryWithTypeCheckers.js 18.6 kB {0} [built]
[1]   [163] ./~/prop-types/lib/ReactPropTypesSecret.js 436 bytes {0} [built]
[1]   [164] ./~/prop-types/checkPropTypes.js 2.94 kB {0} [built]
[1]   [165] ./~/react/lib/ReactVersion.js 350 bytes {0} [built]
[1]   [166] ./~/react/lib/createClass.js 688 bytes {0} [built]
[1]   [167] ./~/create-react-class/factory.js 28.4 kB {0} [built]
[1]   [168] ./~/react/lib/onlyChild.js 1.34 kB {0} [built]
[1]   [169] ./~/react-dom/lib/ReactBrowserEventEmitter.js 12.6 kB {0} [built]
[1]   [170] ./~/react-dom/lib/EventPluginRegistry.js 9.75 kB {0} [built]
[1]   [171] ./~/react-dom/lib/ReactEventEmitterMixin.js 958 bytes {0} [built]
[1]   [172] ./~/react-dom/lib/EventPluginHub.js 9.11 kB {0} [built]
[1]   [173] ./~/react-dom/lib/EventPluginUtils.js 7.95 kB {0} [built]
[1]   [174] ./~/react-dom/lib/ReactErrorUtils.js 2.19 kB {0} [built]
[1]   [175] ./~/react-dom/lib/accumulateInto.js 1.69 kB {0} [built]
[1]   [176] ./~/react-dom/lib/forEachAccumulated.js 855 bytes {0} [built]
[1]   [177] ./~/react-dom/lib/ViewportMetrics.js 604 bytes {0} [built]
[1]   [178] ./~/react-dom/lib/getVendorPrefixedEventName.js 2.87 kB {0} [built]
[1]   [179] ./~/react-dom/lib/isEventSupported.js 1.94 kB {0} [built]
[1]   [180] ./~/react-dom/lib/ReactDOMComponentTree.js 6.27 kB {0} [built]
[1]   [181] ./~/react-dom/lib/ReactDOMComponentFlags.js 429 bytes {0} [built]
[1]   [182] ./~/react-dom/lib/ReactDOMContainerInfo.js 967 bytes {0} [built]
[1]   [183] ./~/react-dom/lib/validateDOMNesting.js 13.7 kB {0} [built]
[1]   [184] ./~/react-dom/lib/ReactDOMFeatureFlags.js 439 bytes {0} [built]
[1]   [185] ./~/react-dom/lib/ReactFeatureFlags.js 628 bytes {0} [built]
[1]   [186] ./~/react-dom/lib/ReactInstanceMap.js 1.22 kB {0} [built]
[1]   [187] ./~/react-dom/lib/ReactInstrumentation.js 601 bytes {0} [built]
[1]   [188] ./~/react-dom/lib/ReactDebugTool.js 11.7 kB {0} [built]
[1]   [189] ./~/react-dom/lib/ReactInvalidSetStateWarningHook.js 984 bytes {0} [built]
[1]   [190] ./~/react-dom/lib/ReactHostOperationHistoryHook.js 736 bytes {0} [built]
[1]   [191] ./~/fbjs/lib/performanceNow.js 845 bytes {0} [built]
[1]   [192] ./~/fbjs/lib/performance.js 588 bytes {0} [built]
[1]   [193] ./~/react-dom/lib/ReactMarkupChecksum.js 1.47 kB {0} [built]
[1]   [194] ./~/react-dom/lib/adler32.js 1.19 kB {0} [built]
[1]   [195] ./~/react-dom/lib/ReactReconciler.js 6.21 kB {0} [built]
[1]   [196] ./~/react-dom/lib/ReactRef.js 2.56 kB {0} [built]
[1]   [197] ./~/react-dom/lib/ReactOwner.js 3.53 kB {0} [built]
[1]   [198] ./~/react-dom/lib/ReactUpdateQueue.js 9.35 kB {0} [built]
[1]   [199] ./~/react-dom/lib/ReactUpdates.js 9.53 kB {0} [built]
[1]   [200] ./~/react-dom/lib/CallbackQueue.js 3.16 kB {0} [built]
[1]   [201] ./~/react-dom/lib/PooledClass.js 3.36 kB {0} [built]
[1]   [202] ./~/react-dom/lib/Transaction.js 9.55 kB {0} [built]
[1]   [203] ./~/react-dom/lib/instantiateReactComponent.js 5.06 kB {0} [built]
[1]   [204] ./~/react-dom/lib/ReactCompositeComponent.js 35.2 kB {0} [built]
[1]   [205] ./~/react-dom/lib/ReactComponentEnvironment.js 1.3 kB {0} [built]
[1]   [206] ./~/react-dom/lib/ReactNodeTypes.js 1.02 kB {0} [built]
[1]   [207] ./~/react-dom/lib/checkReactTypeSpec.js 4.21 kB {0} [built]
[1]   [208] ./~/react-dom/lib/ReactPropTypeLocationNames.js 572 bytes {0} [built]
[1]   [209] ./~/react-dom/lib/ReactPropTypesSecret.js 442 bytes {0} [built]
[1]   [210] ./~/fbjs/lib/shallowEqual.js 1.74 kB {0} [built]
[1]   [211] ./~/react-dom/lib/shouldUpdateReactComponent.js 1.4 kB {0} [built]
[1]   [212] ./~/react-dom/lib/ReactEmptyComponent.js 704 bytes {0} [built]
[1]   [213] ./~/react-dom/lib/ReactHostComponent.js 1.98 kB {0} [built]
[1]   [214] ./~/react/lib/getNextDebugID.js 437 bytes {0} [built]
[1]   [215] ./~/react/react.js 56 bytes {0} [built]
[1]   [217] ./~/minimist/index.js 7.19 kB {0} [built]
[1]   [218] ./browser/src/Config.ts 6.43 kB {0} [built]
[1]   [219] ./~/lodash/lodash.js 539 kB {0} [built]
[1]   [221] ./browser/src/Performance.ts 1.72 kB {0} [built]
[1]   [222] ./~/react-hot-loader/makeExportsHot.js 1.73 kB {0} [built]
[1]   [223] ./~/react-hot-loader/isReactClassish.js 801 bytes {0} [built]
[1]   [224] ./~/react-hot-loader/isReactElementish.js 288 bytes {0} [built]
[1]   [225] ./browser/src/Platform.ts 1.64 kB {0} [built]
[1]   [227] ./browser/src/Plugins/PluginManager.ts 13 kB {0} [built]
[1]   [228] ./~/mkdirp/index.js 2.63 kB {0} [built]
[1]   [229] ./browser/src/Services/CommandManager.ts 2.94 kB {0} [built]
[1]   [230] ./browser/src/UI/index.tsx 3.32 kB {0} [built]
[1]   [231] ./~/react-dom/index.js 59 bytes {0} [built]
[1]   [232] ./~/react-dom/lib/ReactDOM.js 5.17 kB {0} [built]
[1]   [233] ./~/react-dom/lib/ReactDefaultInjection.js 3.5 kB {0} [built]
[1]   [234] ./~/react-dom/lib/ARIADOMPropertyConfig.js 1.82 kB {0} [built]
[1]   [235] ./~/react-dom/lib/BeforeInputEventPlugin.js 13.3 kB {0} [built]
[1]   [236] ./~/react-dom/lib/EventPropagators.js 5.09 kB {0} [built]
[1]   [237] ./~/react-dom/lib/FallbackCompositionState.js 2.43 kB {0} [built]
[1]   [238] ./~/react-dom/lib/getTextContentAccessor.js 955 bytes {0} [built]
[1]   [239] ./~/react-dom/lib/SyntheticCompositionEvent.js 1.1 kB {0} [built]
[1]   [240] ./~/react-dom/lib/SyntheticEvent.js 9.18 kB {0} [built]
[1]   [241] ./~/react-dom/lib/SyntheticInputEvent.js 1.09 kB {0} [built]
[1]   [242] ./~/react-dom/lib/ChangeEventPlugin.js 10.8 kB {0} [built]
[1]   [243] ./~/react-dom/lib/inputValueTracking.js 3.1 kB {0} [built]
[1]   [244] ./~/react-dom/lib/getEventTarget.js 1.01 kB {0} [built]
[1]   [245] ./~/react-dom/lib/isTextInputElement.js 1.02 kB {0} [built]
[1]   [246] ./~/react-dom/lib/DefaultEventPluginOrder.js 1.08 kB {0} [built]
[1]   [247] ./~/react-dom/lib/EnterLeaveEventPlugin.js 3.15 kB {0} [built]
[1]   [248] ./~/react-dom/lib/SyntheticMouseEvent.js 2.14 kB {0} [built]
[1]   [249] ./~/react-dom/lib/SyntheticUIEvent.js 1.57 kB {0} [built]
[1]   [250] ./~/react-dom/lib/getEventModifierState.js 1.23 kB {0} [built]
[1]   [251] ./~/react-dom/lib/HTMLDOMPropertyConfig.js 6.57 kB {0} [built]
[1]   [252] ./~/react-dom/lib/ReactComponentBrowserEnvironment.js 904 bytes {0} [built]
[1]   [253] ./~/react-dom/lib/DOMChildrenOperations.js 7.72 kB {0} [built]
[1]   [254] ./~/react-dom/lib/Danger.js 2.24 kB {0} [built]
[1]   [255] ./~/fbjs/lib/createNodesFromMarkup.js 2.66 kB {0} [built]
[1]   [256] ./~/fbjs/lib/createArrayFromMixed.js 4.11 kB {0} [built]
[1]   [257] ./~/fbjs/lib/getMarkupWrap.js 3.04 kB {0} [built]
[1]   [258] ./~/react-dom/lib/ReactDOMIDOperations.js 955 bytes {0} [built]
[1]   [259] ./~/react-dom/lib/ReactDOMComponent.js 38.9 kB {0} [built]
[1]   [260] ./~/react-dom/lib/AutoFocusUtils.js 599 bytes {0} [built]
[1]   [261] ./~/fbjs/lib/focusNode.js 704 bytes {0} [built]
[1]   [262] ./~/react-dom/lib/CSSPropertyOperations.js 7.2 kB {0} [built]
[1]   [263] ./~/react-dom/lib/CSSProperty.js 3.8 kB {0} [built]
[1]   [264] ./~/fbjs/lib/camelizeStyleName.js 1 kB {0} [built]
[1]   [265] ./~/fbjs/lib/camelize.js 708 bytes {0} [built]
[1]   [266] ./~/react-dom/lib/dangerousStyleValue.js 3.06 kB {0} [built]
[1]   [267] ./~/fbjs/lib/hyphenateStyleName.js 974 bytes {0} [built]
[1]   [268] ./~/fbjs/lib/hyphenate.js 800 bytes {0} [built]
[1]   [269] ./~/fbjs/lib/memoizeStringOnly.js 698 bytes {0} [built]
[1]   [270] ./~/react-dom/lib/DOMPropertyOperations.js 7.61 kB {0} [built]
[1]   [271] ./~/react-dom/lib/quoteAttributeValueForBrowser.js 700 bytes {0} [built]
[1]   [272] ./~/react-dom/lib/ReactDOMInput.js 13.1 kB {0} [built]
[1]   [273] ./~/react-dom/lib/LinkedValueUtils.js 5.24 kB {0} [built]
[1]   [274] ./~/react-dom/lib/ReactDOMOption.js 3.69 kB {0} [built]
[1]   [275] ./~/react-dom/lib/ReactDOMSelect.js 6.81 kB {0} [built]
[1]   [276] ./~/react-dom/lib/ReactDOMTextarea.js 6.46 kB {0} [built]
[1]   [277] ./~/react-dom/lib/ReactMultiChild.js 14.6 kB {0} [built]
[1]   [278] ./~/react-dom/lib/ReactChildReconciler.js 6.11 kB {0} [built]
[1]   [279] ./~/react-dom/lib/KeyEscapeUtils.js 1.29 kB {0} [built]
[1]   [280] ./~/react-dom/lib/traverseAllChildren.js 7.04 kB {0} [built]
[1]   [281] ./~/react-dom/lib/ReactElementSymbol.js 622 bytes {0} [built]
[1]   [282] ./~/react-dom/lib/getIteratorFn.js 1.12 kB {0} [built]
[1]   [283] ./~/react-dom/lib/flattenChildren.js 2.77 kB {0} [built]
[1]   [284] ./~/react-dom/lib/ReactServerRenderingTransaction.js 2.29 kB {0} [built]
[1]   [285] ./~/react-dom/lib/ReactServerUpdateQueue.js 4.83 kB {0} [built]
[1]   [286] ./~/react-dom/lib/ReactDOMEmptyComponent.js 1.9 kB {0} [built]
[1]   [287] ./~/react-dom/lib/ReactDOMTreeTraversal.js 3.72 kB {0} [built]
[1]   [288] ./~/react-dom/lib/ReactDOMTextComponent.js 5.81 kB {0} [built]
[1]   [289] ./~/react-dom/lib/ReactDefaultBatchingStrategy.js 1.88 kB {0} [built]
[1]   [290] ./~/react-dom/lib/ReactEventListener.js 5.3 kB {0} [built]
[1]   [291] ./~/fbjs/lib/EventListener.js 2.37 kB {0} [built]
[1]   [292] ./~/fbjs/lib/getUnboundedScrollPosition.js 1.12 kB {0} [built]
[1]   [293] ./~/react-dom/lib/ReactInjection.js 1.2 kB {0} [built]
[1]   [294] ./~/react-dom/lib/ReactReconcileTransaction.js 5.26 kB {0} [built]
[1]   [295] ./~/react-dom/lib/ReactInputSelection.js 4.27 kB {0} [built]
[1]   [296] ./~/react-dom/lib/ReactDOMSelection.js 6.78 kB {0} [built]
[1]   [297] ./~/react-dom/lib/getNodeForCharacterOffset.js 1.62 kB {0} [built]
[1]   [298] ./~/fbjs/lib/containsNode.js 1.05 kB {0} [built]
[1]   [299] ./~/fbjs/lib/isTextNode.js 605 bytes {0} [built]
[1]   [300] ./~/fbjs/lib/isNode.js 828 bytes {0} [built]
[1]   [301] ./~/fbjs/lib/getActiveElement.js 1.04 kB {0} [built]
[1]   [302] ./~/react-dom/lib/SVGDOMPropertyConfig.js 7.32 kB {0} [built]
[1]   [303] ./~/react-dom/lib/SelectEventPlugin.js 6.06 kB {0} [built]
[1]   [304] ./~/react-dom/lib/SimpleEventPlugin.js 7.97 kB {0} [built]
[1]   [305] ./~/react-dom/lib/SyntheticAnimationEvent.js 1.21 kB {0} [built]
[1]   [306] ./~/react-dom/lib/SyntheticClipboardEvent.js 1.17 kB {0} [built]
[1]   [307] ./~/react-dom/lib/SyntheticFocusEvent.js 1.07 kB {0} [built]
[1]   [308] ./~/react-dom/lib/SyntheticKeyboardEvent.js 2.71 kB {0} [built]
[1]   [309] ./~/react-dom/lib/getEventCharCode.js 1.5 kB {0} [built]
[1]   [310] ./~/react-dom/lib/getEventKey.js 2.87 kB {0} [built]
[1]   [311] ./~/react-dom/lib/SyntheticDragEvent.js 1.07 kB {0} [built]
[1]   [312] ./~/react-dom/lib/SyntheticTouchEvent.js 1.28 kB {0} [built]
[1]   [313] ./~/react-dom/lib/SyntheticTransitionEvent.js 1.23 kB {0} [built]
[1]   [314] ./~/react-dom/lib/SyntheticWheelEvent.js 1.92 kB {0} [built]
[1]   [315] ./~/react-dom/lib/ReactVersion.js 350 bytes {0} [built]
[1]   [316] ./~/react-dom/lib/findDOMNode.js 2.46 kB {0} [built]
[1]   [317] ./~/react-dom/lib/getHostComponentFromComposite.js 740 bytes {0} [built]
[1]   [318] ./~/react-dom/lib/renderSubtreeIntoContainer.js 422 bytes {0} [built]
[1]   [319] ./~/react-dom/lib/ReactDOMUnknownPropertyHook.js 4.32 kB {0} [built]
[1]   [320] ./~/react-dom/lib/ReactDOMNullInputValuePropHook.js 1.37 kB {0} [built]
[1]   [321] ./~/react-dom/lib/ReactDOMInvalidARIAHook.js 3.14 kB {0} [built]
[1]   [322] ./~/react-redux/lib/index.js 751 bytes {0} [built]
[1]   [323] ./~/react-redux/lib/components/Provider.js 3.44 kB {0} [built]
[1]   [324] ./~/prop-types/index.js 1.08 kB {0} [built]
[1]   [325] ./~/react-redux/lib/utils/PropTypes.js 814 bytes {0} [built]
[1]   [326] ./~/react-redux/lib/utils/warning.js 691 bytes {0} [built]
[1]   [327] ./~/react-redux/lib/components/connectAdvanced.js 14 kB {0} [built]
[1]   [328] ./~/hoist-non-react-statics/index.js 1.39 kB {0} [built]
[1]   [329] ./~/invariant/invariant.js 1.54 kB {0} [built]
[1]   [330] ./~/react-redux/lib/utils/Subscription.js 2.63 kB {0} [built]
[1]   [331] ./~/react-redux/lib/connect/connect.js 5.91 kB {0} [built]
[1]   [332] ./~/react-redux/lib/utils/shallowEqual.js 736 bytes {0} [built]
[1]   [333] ./~/react-redux/lib/connect/mapDispatchToProps.js 1.19 kB {0} [built]
[1]   [334] ./~/redux/lib/index.js 1.94 kB {0} [built]
[1]   [335] ./~/redux/lib/createStore.js 9.28 kB {0} [built]
[1]   [336] ./~/lodash/isPlainObject.js 1.65 kB {0} [built]
[1]   [337] ./~/lodash/_baseGetTag.js 809 bytes {0} [built]
[1]   [338] ./~/lodash/_Symbol.js 118 bytes {0} [built]
[1]   [339] ./~/lodash/_root.js 300 bytes {0} [built]
[1]   [340] ./~/lodash/_freeGlobal.js 173 bytes {0} [built]
[1]   [341] ./~/lodash/_getRawTag.js 1.14 kB {0} [built]
[1]   [342] ./~/lodash/_objectToString.js 565 bytes {0} [built]
[1]   [343] ./~/lodash/_getPrototype.js 163 bytes {0} [built]
[1]   [344] ./~/lodash/_overArg.js 382 bytes {0} [built]
[1]   [345] ./~/lodash/isObjectLike.js 614 bytes {0} [built]
[1]   [346] ./~/symbol-observable/index.js 41 bytes {0} [built]
[1]   [347] ./~/symbol-observable/lib/index.js 661 bytes {0} [built]
[1]   [348] ./~/symbol-observable/lib/ponyfill.js 449 bytes {0} [built]
[1]   [349] ./~/redux/lib/combineReducers.js 6.24 kB {0} [built]
[1]   [350] ./~/redux/lib/utils/warning.js 694 bytes {0} [built]
[1]   [351] ./~/redux/lib/bindActionCreators.js 2.04 kB {0} [built]
[1]   [352] ./~/redux/lib/applyMiddleware.js 2.04 kB {0} [built]
[1]   [353] ./~/redux/lib/compose.js 927 bytes {0} [built]
[1]   [354] ./~/react-redux/lib/connect/wrapMapToProps.js 3.17 kB {0} [built]
[1]   [355] ./~/react-redux/lib/utils/verifyPlainObject.js 626 bytes {0} [built]
[1]   [356] ./~/react-redux/lib/connect/mapStateToProps.js 693 bytes {0} [built]
[1]   [357] ./~/react-redux/lib/connect/mergeProps.js 2.04 kB {0} [built]
[1]   [358] ./~/react-redux/lib/connect/selectorFactory.js 4.51 kB {0} [built]
[1]   [359] ./~/react-redux/lib/connect/verifySubselectors.js 997 bytes {0} [built]
[1]   [360] ./~/redux-thunk/lib/index.js 529 bytes {0} [built]
[1]   [361] ./browser/src/UI/RootComponent.tsx 2.78 kB {0} [built]
[1]   [362] ./browser/src/UI/components/Background.tsx 2.63 kB {0} [built]
[1]   [363] ./browser/src/UI/State.ts 3.01 kB {0} [built]
[1]   [364] ./browser/src/UI/components/EditorHost.tsx 1.67 kB {0} [built]
[1]   [365] ./browser/src/UI/components/Logs.tsx 6.01 kB {0} [built]
[1]   [366] ./browser/src/UI/ActionCreators.ts 9.03 kB {0} [built]
[1]   [367] ./browser/src/UI/Events.ts 1.56 kB {0} [built]
[1]   [368] ./browser/src/Utility.ts 1.54 kB {0} [built]
[1]   [369] ./browser/src/UI/Icon.tsx 2.83 kB {0} [built]
[1]   [370] ./browser/src/UI/components/Logs.less 1.09 kB {0} [built]
[1]   [371] ./~/css-loader?-url!./~/less-loader!./browser/src/UI/components/Logs.less 1.3 kB {0} [built]
[1]   [372] ./~/css-loader/lib/css-base.js 1.51 kB {0} [built]
[1]   [373] ./~/style-loader/addStyles.js 7.15 kB {0} [built]
[1]   [374] ./browser/src/UI/components/Menu.tsx 4.89 kB {0} [built]
[1]   [375] ./browser/src/UI/components/HighlightText.tsx 3.46 kB {0} [built]
[1]   [376] ./browser/src/UI/components/Visible.tsx 1.64 kB {0} [built]
[1]   [377] ./browser/src/UI/components/Menu.less 1.09 kB {0} [built]
[1]   [378] ./~/css-loader?-url!./~/less-loader!./browser/src/UI/components/Menu.less 1.92 kB {0} [built]
[1]   [379] ./browser/src/UI/components/StatusBar.tsx 4.12 kB {0} [built]
[1]   [380] ./browser/src/UI/components/StatusBar.less 1.1 kB {0} [built]
[1]   [381] ./~/css-loader?-url!./~/less-loader!./browser/src/UI/components/StatusBar.less 2.15 kB {0} [built]
[1]   [382] ./~/reselect/lib/index.js 4.14 kB {0} [built]
[1]   [383] ./browser/src/UI/Reducer.ts 15.5 kB {0} [built]
[1]   [385] ./~/fuse.js/src/fuse.js 24.2 kB {0} [built]
[1]   [386] ./browser/src/UI/Selectors.ts 3.09 kB {0} [built]
[1]   [387] ./browser/src/Editor/NeovimEditor.tsx 17.2 kB {0} [built]
[1]   [388] ./browser/src/DeltaRegionTracker.ts 2.18 kB {0} [built]
[1]   [389] ./browser/src/Grid.ts 3.9 kB {0} [built]
[1]   [390] ./browser/src/neovim/index.ts 1.66 kB {0} [built]
[1]   [391] ./browser/src/neovim/Buffer.ts 3.67 kB {0} [built]
[1]   [392] ./browser/src/neovim/MsgPack.ts 2.33 kB {0} [built]
[1]   [393] ./~/msgpack-lite/index.js 482 bytes {0} [built]
[1]   [394] ./~/msgpack-lite/lib/encode.js 229 bytes {0} [built]
[1]   [395] ./~/msgpack-lite/lib/encode-buffer.js 629 bytes {0} [built]
[1]   [396] ./~/msgpack-lite/lib/write-core.js 1.7 kB {0} [built]
[1]   [397] ./~/msgpack-lite/lib/ext-buffer.js 260 bytes {0} [built]
[1]   [398] ./~/msgpack-lite/lib/bufferish.js 2.72 kB {0} [built]
[1]   [399] ./~/msgpack-lite/lib/buffer-global.js 230 bytes {0} [built]
[1]   [400] ./~/isarray/index.js 132 bytes {0} [built]
[1]   [401] ./~/msgpack-lite/lib/bufferish-array.js 964 bytes {0} [built]
[1]   [402] ./~/msgpack-lite/lib/bufferish-buffer.js 1.17 kB {0} [built]
[1]   [403] ./~/msgpack-lite/lib/bufferish-uint8array.js 1.37 kB {0} [built]
[1]   [404] ./~/msgpack-lite/lib/bufferish-proto.js 2.26 kB {0} [built]
[1]   [405] ./~/msgpack-lite/lib/buffer-lite.js 3.13 kB {0} [built]
[1]   [406] ./~/msgpack-lite/lib/ext-packer.js 2.52 kB {0} [built]
[1]   [407] ./~/msgpack-lite/lib/write-type.js 7.16 kB {0} [built]
[1]   [408] ./~/int64-buffer/int64-buffer.js 8.42 kB {0} [built]
[1]   [409] ./~/msgpack-lite/lib/write-token.js 6.09 kB {0} [built]
[1]   [410] ./~/ieee754/index.js 2.05 kB {0} [built]
[1]   [411] ./~/msgpack-lite/lib/write-uint8.js 266 bytes {0} [built]
[1]   [412] ./~/msgpack-lite/lib/codec-base.js 1.21 kB {0} [built]
[1]   [413] ./~/msgpack-lite/lib/flex-buffer.js 4.21 kB {0} [built]
[1]   [414] ./~/msgpack-lite/lib/decode.js 228 bytes {0} [built]
[1]   [415] ./~/msgpack-lite/lib/decode-buffer.js 623 bytes {0} [built]
[1]   [416] ./~/msgpack-lite/lib/read-core.js 1.29 kB {0} [built]
[1]   [417] ./~/msgpack-lite/lib/ext-unpacker.js 2.7 kB {0} [built]
[1]   [418] ./~/msgpack-lite/lib/read-format.js 4.51 kB {0} [built]
[1]   [419] ./~/msgpack-lite/lib/read-token.js 3.52 kB {0} [built]
[1]   [420] ./~/msgpack-lite/lib/encoder.js 580 bytes {0} [built]
[1]   [421] ./~/event-lite/event-lite.js 4.23 kB {0} [built]
[1]   [422] ./~/msgpack-lite/lib/decoder.js 621 bytes {0} [built]
[1]   [423] ./~/msgpack-lite/lib/encode-stream.js 906 bytes {0} [built]
[1]   [424] ./~/msgpack-lite/lib/decode-stream.js 820 bytes {0} [built]
[1]   [425] ./~/msgpack-lite/lib/ext.js 144 bytes {0} [built]
[1]   [426] ./~/msgpack-lite/lib/codec.js 185 bytes {0} [built]
[1]   [427] ./browser/src/neovim/Session.ts 3.94 kB {0} [built]
[1]   [428] ./browser/src/neovim/NeovimProcessSpawner.ts 2.71 kB {0} [built]
[1]   [429] ./browser/src/neovim/NeovimInstance.ts 14.4 kB {0} [built]
[1]   [430] ./browser/src/neovim/QuickFix.ts 1.67 kB {0} [built]
[1]   [431] ./browser/src/neovim/Window.ts 2.38 kB {0} [built]
[1]   [432] ./browser/src/actions.ts 5.09 kB {0} [built]
[1]   [433] ./browser/src/Font.ts 2.19 kB {0} [built]
[1]   [434] ./browser/src/Renderer/index.ts 1.53 kB {0} [built]
[1]   [435] ./browser/src/Renderer/CanvasRenderer.ts 9.28 kB {0} [built]
[1]   [436] ./browser/src/Renderer/Span.ts 5.03 kB {0} [built]
[1]   [437] ./browser/src/Renderer/DOMRenderer.ts 5.31 kB {0} [built]
[1]   [438] ./browser/src/Renderer/ElementFactory.ts 3.11 kB {0} [built]
[1]   [439] ./browser/src/Renderer/TokenRenderer.ts 6.02 kB {0} [built]
[1]   [440] ./browser/src/Screen.ts 9.24 kB {0} [built]
[1]   [441] ./~/wcwidth/index.js 3.14 kB {0} [built]
[1]   [442] ./~/defaults/index.js 277 bytes {0} [built]
[1]   [443] ./~/clone/clone.js 4.22 kB {0} [built]
[1]   [444] ./~/wcwidth/combining.js 3.08 kB {0} [built]
[1]   [445] ./browser/src/Services/AutoCompletion.ts 3.41 kB {0} [built]
[1]   [446] ./~/vscode-languageserver-types/lib/main.js 36.1 kB {0} [built]
[1]   [447] ./~/vscode-languageserver-types/lib ^\.\/.*$ 214 bytes {0} [built] [1 warning]
[1]   [449] ./browser/src/Services/AutoCompletionUtility.ts 2.88 kB {0} [built]
[1]   [450] ./browser/src/Services/BufferUpdates.ts 3.12 kB {0} [built]
[1]   [451] ./browser/src/Services/Commands.ts 4.62 kB {0} [built]
[1]   [452] ./browser/src/Services/Errors.ts 2.99 kB {0} [built]
[1]   [453] ./browser/src/Services/Formatter.ts 3.3 kB {0} [built]
[1]   [454] ./browser/src/Services/LiveEvaluation.ts 3.75 kB {0} [built]
[1]   [455] ./browser/src/Services/MultiProcess.ts 1.66 kB {0} [built]
[1]   [456] ./browser/src/Services/Output.ts 4.52 kB {0} [built]
[1]   [457] ./browser/src/Services/QuickOpen.ts 4.47 kB {0} [built]
[1]   [458] ./browser/src/Log.ts 1.95 kB {0} [built]
[1]   [459] ./~/glob/glob.js 19.5 kB {0} [built]
[1]   [460] ./~/fs.realpath/index.js 1.31 kB {0} [built]
[1]   [461] ./~/fs.realpath/old.js 8.54 kB {0} [built]
[1]   [462] ./~/minimatch/minimatch.js 25.3 kB {0} [built]
[1]   [463] ./~/brace-expansion/index.js 4.79 kB {0} [built]
[1]   [464] ./~/concat-map/index.js 345 bytes {0} [built]
[1]   [465] ./~/balanced-match/index.js 1.17 kB {0} [built]
[1]   [467] ./~/path-is-absolute/index.js 611 bytes {0} [built]
[1]   [468] ./~/glob/sync.js 12 kB {0} [built]
[1]   [469] ./~/glob/common.js 6.17 kB {0} [built]
[1]   [470] ./~/inflight/inflight.js 1.36 kB {0} [built]
[1]   [471] ./~/wrappy/wrappy.js 905 bytes {0} [built]
[1]   [472] ./~/once/once.js 935 bytes {0} [built]
[1]   [473] ./~/q/q.js 62.8 kB {0} [built]
[1]   [474] ./browser/src/PromiseHelper.ts 1.77 kB {0} [built]
[1]   [475] ./browser/src/Services/Git.ts 2.75 kB {0} [built]
[1]   [476] ./browser/src/Services/SyntaxHighlighter.ts 4.66 kB {0} [built]
[1]   [477] ./browser/src/Services/Tasks.ts 5.6 kB {0} [built]
[1]   [478] ./browser/src/Parser.ts 1.52 kB {0} [built]
[1]   [479] ./browser/src/ProjectConfig.ts 2.26 kB {0} [built]
[1]   [480] ./~/find-up/index.js 1.01 kB {0} [built]
[1]   [481] ./~/locate-path/index.js 512 bytes {0} [built]
[1]   [482] ./~/path-exists/index.js 252 bytes {0} [built]
[1]   [483] ./~/p-locate/index.js 934 bytes {0} [built]
[1]   [484] ./~/p-limit/index.js 597 bytes {0} [built]
[1]   [485] ./browser/src/Services/WindowTitle.ts 1.59 kB {0} [built]
[1]   [486] ./browser/src/UI/Overlay/LiveEvaluationOverlay.ts 2.66 kB {0} [built]
[1]   [487] ./browser/src/UI/components/LiveEvalMarker.tsx 3.76 kB {0} [built]
[1]   [488] ./~/classnames/index.js 1.1 kB {0} [built]
[1]   [489] ./browser/src/UI/components/LiveEvalMarker.less 1.12 kB {0} [built]
[1]   [490] ./~/css-loader?-url!./~/less-loader!./browser/src/UI/components/LiveEvalMarker.less 647 bytes {0} [built]
[1]   [491] ./browser/src/UI/Overlay/OverlayManager.ts 4.33 kB {0} [built]
[1]   [492] ./browser/src/UI/Overlay/WindowContext.ts 4.89 kB {0} [built]
[1]   [493] ./browser/src/Input/Keyboard.ts 3.17 kB {0} [built]
[1]   [494] ./browser/src/UI/components/InstallHelp.tsx 2.44 kB {0} [built]
[1]   [495] ./browser/src/UI/components/InstallHelp.less 1.11 kB {0} [built]
[1]   [496] ./~/css-loader?-url!./~/less-loader!./browser/src/UI/components/InstallHelp.less 747 bytes {0} [built]
[1]   [497] ./browser/src/Editor/NeovimSurface.tsx 3.86 kB {0} [built]
[1]   [498] ./browser/src/UI/components/ActiveWindow.tsx 2.2 kB {0} [built]
[1]   [499] ./browser/src/UI/components/AutoCompletion.tsx 7.53 kB {0} [built]
[1]   [500] ./browser/src/UI/components/AutoCompletion.less 1.12 kB {0} [built]
[1]   [501] ./~/css-loader?-url!./~/less-loader!./browser/src/UI/components/AutoCompletion.less 2.07 kB {0} [built]
[1]   [502] ./browser/src/UI/components/BufferScrollBar.tsx 4.51 kB {0} [built]
[1]   [503] ./browser/src/UI/components/BufferScrollBar.less 1.12 kB {0} [built]
[1]   [504] ./~/css-loader?-url!./~/less-loader!./browser/src/UI/components/BufferScrollBar.less 582 bytes {0} [built]
[1]   [505] ./browser/src/UI/components/Cursor.tsx 2.88 kB {0} [built]
[1]   [506] ./browser/src/UI/components/Cursor.less 1.09 kB {0} [built]
[1]   [507] ./~/css-loader?-url!./~/less-loader!./browser/src/UI/components/Cursor.less 279 bytes {0} [built]
[1]   [508] ./browser/src/UI/components/CursorLine.tsx 3.25 kB {0} [built]
[1]   [509] ./browser/src/UI/components/Error.tsx 6.88 kB {0} [built]
[1]   [510] ./browser/src/UI/components/Error.less 1.09 kB {0} [built]
[1]   [511] ./~/css-loader?-url!./~/less-loader!./browser/src/UI/components/Error.less 1.43 kB {0} [built]
[1]   [512] ./browser/src/UI/components/QuickInfo.tsx 6.9 kB {0} [built]
[1]   [513] ./browser/src/UI/components/QuickInfo.less 1.1 kB {0} [built]
[1]   [514] ./~/css-loader?-url!./~/less-loader!./browser/src/UI/components/QuickInfo.less 1.05 kB {0} [built]
[1]   [515] ./browser/src/UI/components/Tabs.tsx 4.95 kB {0} [built]
[1]   [516] ./browser/src/UI/components/Tabs.less 1.09 kB {0} [built]
[1]   [517] ./~/css-loader?-url!./~/less-loader!./browser/src/UI/components/Tabs.less 2.08 kB {0} [built]
[1]   [518] ./browser/src/Editor/NeovimInput.tsx 2.03 kB {0} [built]
[1]   [519] ./browser/src/Input/Mouse.ts 3.41 kB {0} [built]
[1]   [520] ./browser/src/Editor/NeovimRenderer.tsx 2.71 kB {0} [built]
[1]   [521] ./browser/src/UI/components/common.less 1.09 kB {0} [built]
[1]   [522] ./~/css-loader?-url!./~/less-loader!./browser/src/UI/components/common.less 1.92 kB {0} [built]
[1]   [523] ./~/react-addons-perf/index.js 52 bytes {0} [built]
[1]   [524] ./~/react-dom/lib/ReactPerf.js 14.8 kB {0} [built]
[1]   [525] ./~/react-dom/lib/lowPriorityWarning.js 2.16 kB {0} [built]
[1]   [526] ./browser/src/Plugins/AnonymousPlugin.ts 1.91 kB {0} [built]
[1]   [527] ./browser/src/Plugins/Api/Oni.ts 9.88 kB {0} [built]
[1]   [528] ./browser/src/Plugins/Api/Commands.ts 2.02 kB {0} [built]
[1]   [529] ./browser/src/Plugins/Api/Configuration.ts 1.97 kB {0} [built]
[1]   [530] ./browser/src/Event.ts 1.87 kB {0} [built]
[1]   [531] ./browser/src/Plugins/Api/Diagnostics.ts 2.06 kB {0} [built]
[1]   [532] ./browser/src/Plugins/Api/Editor.ts 1.62 kB {0} [built]
[1]   [533] ./browser/src/Plugins/Api/StatusBar.ts 3 kB {0} [built]
[1]   [534] ./browser/src/Plugins/Api/DebouncedLanguageService.ts 5.21 kB {0} [built]
[1]   [535] ./browser/src/Plugins/Api/PromiseDebouncer.ts 2.88 kB {0} [built]
[1]   [536] ./browser/src/Plugins/Api/LanguageClient/LanguageClient.ts 17.8 kB {0} [built]
[1]   [537] ./~/vscode-jsonrpc/lib/main.js 36.6 kB {0} [built]
[1]   [538] ./~/vscode-jsonrpc/lib/is.js 1.26 kB {0} [built]
[1]   [539] ./~/vscode-jsonrpc/lib/messages.js 11.8 kB {0} [built]
[1]   [540] ./~/vscode-jsonrpc/lib/messageReader.js 10.7 kB {0} [built]
[1]   [541] ./~/vscode-jsonrpc/lib/events.js 6.94 kB {0} [built]
[1]   [542] ./~/vscode-jsonrpc/lib/messageWriter.js 8.44 kB {0} [built]
[1]   [543] ./~/vscode-jsonrpc/lib/cancellation.js 3.63 kB {0} [built]
[1]   [544] ./~/vscode-jsonrpc/lib/linkedMap.js 9.37 kB {0} [built]
[1]   [545] ./~/vscode-jsonrpc/lib/pipeSupport.js 2.39 kB {0} [built]
[1]   [546] ./browser/src/Plugins/Api/LanguageClient/LanguageClientHelpers.ts 5.01 kB {0} [built]
[1]   [547] ./browser/src/Plugins/Api/LanguageClient/LanguageClientLogger.ts 1.68 kB {0} [built]
[1]   [548] ./browser/src/Plugins/Api/LanguageClient/LanguageClientStatusBar.tsx 4.24 kB {0} [built]
[1]   [549] ./browser/src/Plugins/Api/Process.ts 3.25 kB {0} [built]
[1]   [550] ./browser/src/Plugins/Api/Services.ts 1.49 kB {0} [built]
[1]   [551] ./browser/src/Plugins/Api/Ui.ts 1.64 kB {0} [built]
[1]   [552] ./browser/src/Plugins/Api/Capabilities.ts 2.36 kB {0} [built]
[1]   [553] ./browser/src/Plugins/Api/Channel.ts 4.15 kB {0} [built]
[1]   [554] ./browser/src/Plugins/Plugin.ts 3.29 kB {0} [built]
[1]   [555] ./browser/src/Plugins/PackageMetadataParser.ts 2.28 kB {0} [built]
[1]   [557] ./browser/src/overlay.less 1.05 kB {0} [built]
[1]   [558] ./~/css-loader?-url!./~/less-loader!./browser/src/overlay.less 497 bytes {0} [built]
[1]      + 17 hidden modules
[1] 
[1] WARNING in ./~/vscode-languageserver-types/lib/main.js
[1] Critical dependencies:
[1] 3:24-31 require function is used in a way in which dependencies cannot be statically extracted
[1]  @ ./~/vscode-languageserver-types/lib/main.js 3:24-31
[1] 
[1] WARNING in ./~/vscode-languageserver-types/lib/main.d.ts
[1] Module parse failed: /home/hoschi/repos/oni-master/node_modules/vscode-languageserver-types/lib/main.d.ts Unexpected token (4:7)
[1] You may need an appropriate loader to handle this file type.
[1] SyntaxError: Unexpected token (4:7)
[1]     at Parser.pp$4.raise (/home/hoschi/repos/oni-master/node_modules/webpack/node_modules/acorn/dist/acorn.js:2221:15)
[1]     at Parser.pp.unexpected (/home/hoschi/repos/oni-master/node_modules/webpack/node_modules/acorn/dist/acorn.js:603:10)
[1]     at Parser.pp.expect (/home/hoschi/repos/oni-master/node_modules/webpack/node_modules/acorn/dist/acorn.js:597:28)
[1]     at Parser.pp$1.parseExportSpecifiers (/home/hoschi/repos/oni-master/node_modules/webpack/node_modules/acorn/dist/acorn.js:1215:10)
[1]     at Parser.pp$1.parseExport (/home/hoschi/repos/oni-master/node_modules/webpack/node_modules/acorn/dist/acorn.js:1186:30)
[1]     at Parser.pp$1.parseStatement (/home/hoschi/repos/oni-master/node_modules/webpack/node_modules/acorn/dist/acorn.js:719:71)
[1]     at Parser.pp$1.parseTopLevel (/home/hoschi/repos/oni-master/node_modules/webpack/node_modules/acorn/dist/acorn.js:638:25)
[1]     at Parser.parse (/home/hoschi/repos/oni-master/node_modules/webpack/node_modules/acorn/dist/acorn.js:516:17)
[1]     at Object.parse (/home/hoschi/repos/oni-master/node_modules/webpack/node_modules/acorn/dist/acorn.js:3098:39)
[1]     at Parser.parse (/home/hoschi/repos/oni-master/node_modules/webpack/lib/Parser.js:902:15)
[1]  @ ./~/vscode-languageserver-types/lib ^\.\/.*$
[1] webpack: Compiled with warnings.
[0] Attempting to call a function in a renderer window that has been closed or released.
[0] Function provided here: bundle.js:56276:24
[0] Attempting to call a function in a renderer window that has been closed or released.
[0] Function provided here: bundle.js:56279:24
[0] Attempting to call a function in a renderer window that has been closed or released.
[0] Function provided here: bundle.js:56276:24
[0] Attempting to call a function in a renderer window that has been closed or released.
[0] Function provided here: bundle.js:56279:24
[0] Attempting to call a function in a renderer window that has been closed or released.
[0] Function provided here: bundle.js:56276:24
[0] Attempting to call a function in a renderer window that has been closed or released.
[0] Function provided here: bundle.js:56279:24
[0] npm run start-hot exited with code 0
--> Sending SIGTERM to other processes..
[2] npm run watch:plugins exited with code null
[1] npm run watch:browser exited with code null

npm ERR! Linux 4.11.9-1-ARCH
npm ERR! argv "/home/hoschi/.nvm/versions/node/v6.3.1/bin/node" "/home/hoschi/.nvm/versions/node/v6.3.1/bin/npm" "run" "start"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! oni@0.2.8 start: `concurrently --kill-others "npm run start-hot" "npm run watch:browser" "npm run watch:plugins"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the oni@0.2.8 start script 'concurrently --kill-others "npm run start-hot" "npm run watch:browser" "npm run watch:plugins"'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the oni package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     concurrently --kill-others "npm run start-hot" "npm run watch:browser" "npm run watch:plugins"
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs oni
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls oni
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/hoschi/repos/oni-master/npm-debug.log
```

--- a/README.md
+++ b/README.md
@@ -246,7 +246,7 @@ Oni features a buffer tab bar, like many common IDEs. VIM has its own definition
 ##### Options
 
 - `tabs.enabled` - If set to `true`, the tabs are visible. (Default: `true`)
-- `tabs.showVimTabs` - If set to `true`, shows vim tabs. Otherwise, shows open buffers. (Default: `false`)
+- `tabs.showVimTabs` - If set to `true`, shows vim tabs. Otherwise, shows open buffers. (Default: `false`. Requires Neovim 0.2.1+)
 
 ### Languages
 

--- a/browser/src/Config.ts
+++ b/browser/src/Config.ts
@@ -157,7 +157,7 @@ export class Config extends EventEmitter {
         "statusbar.fontSize": "12px",
 
         "tabs.enabled": true,
-        "tabs.showVimTabs": true,
+        "tabs.showVimTabs": false,
     }
 
     private MacConfig: Partial<IConfigValues> = {

--- a/browser/src/neovim/NeovimInstance.ts
+++ b/browser/src/neovim/NeovimInstance.ts
@@ -65,6 +65,7 @@ export interface INeovimInstance {
 export interface INeovimApiVersion {
     major: number
     minor: number
+    patch: number
 }
 
 /**
@@ -208,19 +209,19 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
 
     private async _attachUI(columns: number, rows: number): Promise<void> {
         const version = await this.getApiVersion()
-        const startupOptions = this._getStartupOptionsForVersion(version.major, version.minor)
+        const startupOptions = this._getStartupOptionsForVersion(version.major, version.minor, version.patch)
 
         await this._neovim.request("nvim_ui_attach", [columns, rows, startupOptions])
     }
 
-    private _getStartupOptionsForVersion(major: number, minor: number) {
-        if (major >= 0 && minor >= 2) {
+    private _getStartupOptionsForVersion(major: number, minor: number, patch: number) {
+        if (major >= 0 && minor >= 2 && patch >= 1) {
             return {
                 rgb: true,
                 popupmenu_external: true,
                 ext_tabline: true,
             }
-        } else if (major === 0 && minor === 1) {
+        } else if (major === 0 && minor === 2) {
             // 0.1 and below does not support external tabline
             // See #579 for more info on the manifestation.
             return {

--- a/browser/src/neovim/NeovimInstance.ts
+++ b/browser/src/neovim/NeovimInstance.ts
@@ -436,7 +436,7 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
 
     private async _attachUI(columns: number, rows: number): Promise<void> {
         const version = await this.getApiVersion()
-        console.log(`Neovim version reported as ${version.major}.${version.minor}.${version.patch}`) // tslint:disable-line no-console-log
+        console.log(`Neovim version reported as ${version.major}.${version.minor}.${version.patch}`) // tslint:disable-line no-console
 
         const startupOptions = this._getStartupOptionsForVersion(version.major, version.minor, version.patch)
 

--- a/browser/src/neovim/NeovimInstance.ts
+++ b/browser/src/neovim/NeovimInstance.ts
@@ -57,7 +57,14 @@ export interface INeovimInstance {
     getCursorColumn(): Promise<number>
     getCursorRow(): Promise<number>
 
+    getApiVersion(): Promise<INeovimApiVersion>
+
     open(fileName: string): Promise<void>
+}
+
+export interface INeovimApiVersion {
+    major: number
+    minor: number
 }
 
 /**
@@ -175,21 +182,13 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
                     remote.getCurrentWindow().close()
                 })
 
-                const startupOptions = {
-                    rgb: true,
-                    popupmenu_external: true,
-                    ext_tabline: true,
-                }
-
                 const size = this._getSize()
                 this._rows = size.rows
                 this._cols = size.cols
 
                 // Workaround for bug in neovim/node-client
                 // The 'uiAttach' method overrides the new 'nvim_ui_attach' method
-                return this._neovim.request("nvim_get_api_info", [])
-                    .then((info) => { debugger })
-                    .then(() => this._neovim.request("nvim_ui_attach", [size.cols, size.rows, startupOptions]))
+                return this._attachUI(size.cols, size.rows)
                     .then(() => {
                         this.emit("logInfo", "Attach success")
 
@@ -207,6 +206,32 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
             })
     }
 
+    private async _attachUI(columns: number, rows: number): Promise<void> {
+        const version = await this.getApiVersion()
+        const startupOptions = this._getStartupOptionsForVersion(version.major, version.minor)
+
+        await this._neovim.request("nvim_ui_attach", [columns, rows, startupOptions])
+    }
+
+    private _getStartupOptionsForVersion(major: number, minor: number) {
+        if (major >= 0 && minor >= 2) {
+            return {
+                rgb: true,
+                popupmenu_external: true,
+                ext_tabline: true,
+            }
+        } else if (major === 0 && minor === 1) {
+            // 0.1 and below does not support external tabline
+            // See #579 for more info on the manifestation.
+            return {
+                rgb: true,
+                popupmenu_external: true,
+            }
+        } else {
+            throw "Unsupported version of Neovim."
+        }
+    }
+
     public getMode(): Promise<string> {
         return this.eval<string>("mode()")
     }
@@ -218,8 +243,7 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
         return this.eval<number>("col('.')")
     }
 
-    /**
-     * Returns the current cursor row in buffer-space
+    /** Returns the current cursor row in buffer-space
      */
     public getCursorRow(): Promise<number> {
         return this.eval<number>("line('.')")
@@ -302,6 +326,11 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
         const size = this._getSize()
 
         this._resizeInternal(size.rows, size.cols)
+    }
+
+    public async getApiVersion(): Promise<INeovimApiVersion> {
+        const versionInfo = await this._neovim.request("nvim_get_api_info", [])
+        return <any>versionInfo[1].version
     }
 
     private _resizeInternal(rows: number, columns: number): void {

--- a/browser/src/neovim/NeovimInstance.ts
+++ b/browser/src/neovim/NeovimInstance.ts
@@ -187,7 +187,9 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
 
                 // Workaround for bug in neovim/node-client
                 // The 'uiAttach' method overrides the new 'nvim_ui_attach' method
-                return this._neovim.request("nvim_ui_attach", [size.cols, size.rows, startupOptions])
+                return this._neovim.request("nvim_get_api_info", [])
+                    .then((info) => { debugger })
+                    .then(() => this._neovim.request("nvim_ui_attach", [size.cols, size.rows, startupOptions]))
                     .then(() => {
                         this.emit("logInfo", "Attach success")
 

--- a/browser/src/neovim/NeovimInstance.ts
+++ b/browser/src/neovim/NeovimInstance.ts
@@ -209,6 +209,8 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
 
     private async _attachUI(columns: number, rows: number): Promise<void> {
         const version = await this.getApiVersion()
+        console.log(`Neovim version reported as ${version.major}.${version.minor}.${version.patch}`)
+
         const startupOptions = this._getStartupOptionsForVersion(version.major, version.minor, version.patch)
 
         await this._neovim.request("nvim_ui_attach", [columns, rows, startupOptions])

--- a/browser/src/neovim/NeovimInstance.ts
+++ b/browser/src/neovim/NeovimInstance.ts
@@ -207,34 +207,6 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
             })
     }
 
-    private async _attachUI(columns: number, rows: number): Promise<void> {
-        const version = await this.getApiVersion()
-        console.log(`Neovim version reported as ${version.major}.${version.minor}.${version.patch}`)
-
-        const startupOptions = this._getStartupOptionsForVersion(version.major, version.minor, version.patch)
-
-        await this._neovim.request("nvim_ui_attach", [columns, rows, startupOptions])
-    }
-
-    private _getStartupOptionsForVersion(major: number, minor: number, patch: number) {
-        if (major >= 0 && minor >= 2 && patch >= 1) {
-            return {
-                rgb: true,
-                popupmenu_external: true,
-                ext_tabline: true,
-            }
-        } else if (major === 0 && minor === 2) {
-            // 0.1 and below does not support external tabline
-            // See #579 for more info on the manifestation.
-            return {
-                rgb: true,
-                popupmenu_external: true,
-            }
-        } else {
-            throw "Unsupported version of Neovim."
-        }
-    }
-
     public getMode(): Promise<string> {
         return this.eval<string>("mode()")
     }
@@ -462,4 +434,31 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
         this.emit("directory-changed", newDirectory)
     }
 
+    private async _attachUI(columns: number, rows: number): Promise<void> {
+        const version = await this.getApiVersion()
+        console.log(`Neovim version reported as ${version.major}.${version.minor}.${version.patch}`) // tslint:disable-line no-console-log
+
+        const startupOptions = this._getStartupOptionsForVersion(version.major, version.minor, version.patch)
+
+        await this._neovim.request("nvim_ui_attach", [columns, rows, startupOptions])
+    }
+
+    private _getStartupOptionsForVersion(major: number, minor: number, patch: number) {
+        if (major >= 0 && minor >= 2 && patch >= 1) {
+            return {
+                rgb: true,
+                popupmenu_external: true,
+                ext_tabline: true,
+            }
+        } else if (major === 0 && minor === 2) {
+            // 0.1 and below does not support external tabline
+            // See #579 for more info on the manifestation.
+            return {
+                rgb: true,
+                popupmenu_external: true,
+            }
+        } else {
+            throw "Unsupported version of Neovim."
+        }
+    }
 }

